src/Features/Auth/ducks.js

```diff
@@ -1,41 +1,21 @@
-import {
-  LOGIN,
-  LOGIN_SUCCESS,
-  LOGOUT,
-  LOGIN_ERROR,
-  SIGNUP,
-  SIGNUP_ERROR,
-  SIGNUP_SUCCESS,
-} from '../../scripts/constants';
+import { LOGIN, LOGIN_SUCCESS, LOGOUT, LOGIN_ERROR, SIGNUP } from '../../scripts/constants';
 
-export const loginReducer = (state = { isLoggedIn: false }, action) => {
+export const loginReducer = (state = { isLoggedIn: false, token: '', name: '' }, action) => {
```

Comment:
As we are handling the whole auth logic in this reducer that name must be changed to auth reducer